pallets/protos/src/lib.rs

```diff
@@ -797,7 +797,7 @@ pub mod pallet {
 
 			// make sure the user has enough tickets
 			let balance = <pallet_assets::Pallet<T> as Inspect<T::AccountId>>::balance(
-				T::TicketsAssetId::get(),
+				<T as pallet::Config>::TicketsAssetId::get(),
```

Comment:
Does `<T as pallet::Config>::...` change anything compared to `T::...` or does it not compile without?

Reply:
Uh I need to check again why i did this..

Reply:
this is actually not part of this PR.